bin/compare-factory-config.py

```diff
@@ -90,7 +90,13 @@ def get_gfactory_data(gfactory_DB, filename):
                             print(attr.get('value'))
                         # gfactory structure: {GLIDEIN_ResourceName: entry name, ...}
                         # print(entry.get('name'), 'is',entry.get('enabled'))
-                        gfactory_DB[attr.get('value')] = entry.get('name')
+                        try:
+                            gfactory_DB[attr.get('value')].append(
+                                entry.get('name'))
+                        except:
```

Comment:
You shouldn't catch all exceptions, just the ones you need so you could do this:

```suggestion
                        except KeyError:
```

Also, this is what `collections.defaultdict` is for :)